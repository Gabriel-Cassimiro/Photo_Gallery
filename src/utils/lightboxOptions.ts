export const Options = {
	settings: {},
	caption: {},
	buttons: {},
	thumbnails: {},
	progressBar: {}
}
