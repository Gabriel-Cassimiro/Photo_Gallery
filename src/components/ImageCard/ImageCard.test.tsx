/**
 * @jest-environment jsdom
 */

import { render } from "@testing-library/react"
import { TestProvider } from "../../tests/utils"

import { ImageCard, ImageCardProps } from "./index"
import { photo } from "../../tests/mocks/image.mock"

describe("Test ImageCard component", () => {
	function Element({ image }: ImageCardProps) {
		return (
			<TestProvider>
				<ImageCard image={image} />
			</TestProvider>
		)
	}

	it("Should render correctly", () => {
		const { container } = render(<Element image={photo} />)
		expect(container).toMatchSnapshot()
	})
})
