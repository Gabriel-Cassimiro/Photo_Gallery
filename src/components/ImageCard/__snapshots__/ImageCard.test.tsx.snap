// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ImageCard component Should render correctly 1`] = `
<div>
  <a
    href="https://www.pexels.com/@joey"
  >
    <span
      style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
    >
      <span
        style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
      >
        <img
          alt=""
          aria-hidden="true"
          src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzAyNCIgaGVpZ2h0PSIzMDI0IiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIvPg=="
          style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
        />
      </span>
      <img
        alt="Photo by Joey Farina"
        class="w-full cursor-pointer bg-teal-300 transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-110"
        data-nimg="intrinsic"
        decoding="async"
        src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
        style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
        title="Click Photo to open Gallery"
      />
      <noscript />
    </span>
  </a>
</div>
`;
